webapp_server/chart/cosmotech-business-webapp/values.yaml

```diff
@@ -0,0 +1,30 @@
+# name is the name of the deployment, ingress and service that will be created
+name: cosmotech-business-webapp
+# imageName is the name of the docker image containing your webapp server (e.g. ghcr.io/cosmo-tech/azure-sample-webapp)
+imageName: ""
+# imageTag is the tag of the docker image to pull
+imageTag: latest
+# functionsImageName is the name of the docker image containing your API functions server
+functionsImageName: ""
```

Comment:
Same here, if we go for a structured field for `image` it should be the same for `functionsImage`
And not sure about the rest, maybe `powerbi` can be a group too.